packages/app-builder-lib/src/electron/electronMac.ts

```diff
@@ -121,7 +121,7 @@ export async function createMacApp(packager: MacPackager, appOutDir: string, asa
     helper.CFBundleDisplayName = `${appInfo.productName} Helper ${postfix}`
     helper.CFBundleIdentifier = userProvidedBundleIdentifier
       ? filterCFBundleIdentifier(userProvidedBundleIdentifier)
-      : `${helperBundleIdentifier}.${postfix.replace(/[^a-z0-9]/gim, "")}`
+      : filterCFBundleIdentifier(`${helperBundleIdentifier}.${postfix}`)
```

Comment:
What's the reason for this change?

Reply:
https://github.com/electron-userland/electron-builder/pull/4154
In this pr, I didn't realize that this filterCFBundleIdentifier function existed at that time, now I cleaned it up and replaced it with calling this filterCFBundleIdentifier function to avoid duplicate code

Reply:
@develar  any news?